Sources/UIKitExtensions/UIScrollView+FunctionConfigurationSet.swift

```diff
@@ -0,0 +1,26 @@
+//
+//  UIScrollViewFunctions.swift
+//  ViewConfigurator-iOS
+//
+//  Created by Andrii Pazynych on 27.03.18.
+//  Copyright © 2018 ImagineOn GmbH. All rights reserved.
+//
+
+
+import UIKit
+public extension ConfigurationSet where Base: UIScrollView {
+    
+    func setContentOffset(_ contentOffset: CGPoint, animated: Bool) -> ConfigurationSet<Base> {
+        return set { ( UIScrollView: Base ) in
+            UIScrollView.setContentOffset(contentOffset, animated: animated)
```

Comment:
This function is only useful for existing UIScrollView whose contentOffset change should be animated.
The contentOffset Property ist writable directly so there is no need for this function.